index.html

```diff
@@ -243,6 +243,57 @@ <h3>Skills</h3>
         </div>
       </div>      
     </section>
+    <!-- contact us section -->
+    <section id="contact">
+      <div class="contact-container">
+        <p class="contact-intro">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <!-- form -->
+        <form action="https://formspree.io/f/xpzedlbr" method="POST">
+          
+          <input type="text" name="username" id="name" placeholder="Full name" maxlength="30" required>
+  
+          <input type="email" name="email" id="email" placeholder="Email address" required>
+  
+          <textarea name="message" id="message" placeholder="Enter text here" maxlength="500"></textarea>
```

Comment:
- [ ] we are missing the required attribute in the textarea element, this makes it possible to send empty messages, please fix this